<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-1</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
background-color:rgb(234, 234, 234);}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>80</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 32pt &quot;Calibri&quot;;color:black</string>
    </property>
    <property name="text">
     <string>Vaccination</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submit_vaccination">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>680</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;background-color:rgb(225, 134, 127);
font: 24pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>150</y>
      <width>431</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Calibri&quot;;color:black</string>
    </property>
    <property name="text">
     <string>Please enter the required details below.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_error">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>460</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Calibri&quot;;color:red</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="q1_1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>571</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Calibri&quot;;color:black</string>
    </property>
    <property name="text">
     <string>1. Are you interested to take the COVID-19 vaccine?</string>
    </property>
   </widget>
   <widget class="QLabel" name="q1_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>541</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Calibri&quot;;color:black</string>
    </property>
    <property name="text">
     <string>2. Are you currently pregnant?</string>
    </property>
   </widget>
   <widget class="QLabel" name="q1_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>541</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Calibri&quot;;color:black</string>
    </property>
    <property name="text">
     <string>3.  Are you currently breastfeeding?</string>
    </property>
   </widget>
   <widget class="QLabel" name="q1_5">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>230</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Calibri&quot;;color:black</string>
    </property>
    <property name="text">
     <string>5. Are you a frontliner?</string>
    </property>
   </widget>
   <widget class="QLabel" name="q1_6">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>310</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Calibri&quot;;color:black</string>
    </property>
    <property name="text">
     <string>6. What is your occupation?</string>
    </property>
   </widget>
   <widget class="QLabel" name="q1_7">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>450</y>
      <width>541</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Calibri&quot;;color:black</string>
    </property>
    <property name="text">
     <string>7. Do you have any critical illnesses?</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_q1_1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Calibri&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>No</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Yes</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_q1_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Calibri&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>No</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Yes</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_q1_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>430</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Calibri&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>No</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Yes</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_q1_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>580</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Calibri&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>No</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Yes</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_q1_5">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>270</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Calibri&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>No</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Yes</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_q1_7">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>490</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Calibri&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>No</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Yes</string>
     </property>
    </item>
   </widget>
   <widget class="QTextBrowser" name="q1_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>480</y>
      <width>511</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Calibri&quot;;color:black;background-color:rgba(0,0,0,0);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Calibri'; font-size:16pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:14pt;&quot;&gt;4. Are you registered with the department of Social Welfare Malaysia as a Disabled Person (OKU)?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_1">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>350</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
font: 10pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Health-care worker</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>380</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
font: 10pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Community Services</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>410</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
font: 10pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Workers</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_4">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>350</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
font: 10pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Students</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_5">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>380</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
font: 10pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Other</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_goToMainMenu">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>680</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;background-color:rgb(225, 134, 127);
font: 24pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
